app/controllers/people_controller.rb

```diff
@@ -1,6 +1,8 @@
 class PeopleController < ApplicationController
   before_action :admin_required
-  layout "admin"
+
+ layout "admin"
+
```

Comment:
Proszę nie psuć formatowania. To bardzo utrudnia czytanie kodu. Tym bardziej, że w tym miejscu nic Pani nie dodała/usunęła.